pkg/boot/systembooter/bootbooter.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package systembooter
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+)
+
+// Boot implements the Booter interface for booting from local storage.
+type BootBooter struct {
+	Type         string `json:"type"`
+	DeviceIgnore string `json:"device_ignore"`
+	KernelAppend string `json:"kernel_append"`
+	KernelRemove string `json:"kernel_remove"`
+	KernelReuse  string `json:"kernel_reuse"`
+}
+
+// NewBootBooter parses a boot entry config and returns a Booter instance, or
+// an error if any
+func NewBootBooter(config []byte) (Booter, error) {
+	/* The configuration format for a Boot Booter entry is a JSON with the following structure:
+
+		{
+			"type": "boot",
+			"device_ignore": "<pci devices or empty>",
+			"kernel_append": "<kernel args to append or empty>",
+			"kernel_remove": "<kernel args to remove or empty>",
+			"kernel_reuse":  "<kernel args to reuse or empty>",
+		}
+
+	The JSON corisponds to the boot application command line variables:
+	 "block": comma separated list of pci vendor and device ids to ignore (format vendor:device). E.g. 0x8086:0x1234,0x8086:0xabcd
+	 "append": comma separated list of kernel params value to reuse from current kernel configuration
+	 "remove": comma separated list of kernel params value to remove from parsed kernel configuration
+	 "reuse": comma separated list of kernel params value to reuse from current kernel configuration
+	*/
+
+	log.Printf("Trying Boot Booter...")
```

Comment:
we prefer not to have prints like this in packages, you can if you want pass a logger as the second parameter. 
Also, usually, because you are in the systembooter package, the name would be just New(...). If you use New, will that be a problem?

Reply:
Still need to use function NewBootBooter, because it is the systembooter pkg design:
https://github.com/u-root/u-root/tree/main/pkg/boot/systembooter#creating-a-new-booter